onnx/test/cpp/onnxifi_backend_test.cc

```diff
@@ -0,0 +1,29 @@
+#include <iostream>
+
+#include "gtest/gtest.h"
+#include "onnx/onnxifi_loader.h"
+#include "onnx/onnxifi.h"
+
+#if defined(__APPLE__)
+#define ONNXIFI_DUMMY_LIBRARY "libonnxifi_dummy.dylib"
+#elif defined(_WIN32)
+#define ONNXIFI_DUMMY_LIBRARY L"onnxifi_dummy.dll"
+#else
+#define ONNXIFI_DUMMY_LIBRARY "libonnxifi_dummy.so"
+#endif
+
+namespace ONNX_NAMESPACE
+{
+	namespace Test
+	{
+		TEST(OnnxifiLoadTest, OnnxifiDummyBackend)
+		{
+			onnxifi_library dummy_backend;
+			EXPECT_TRUE(onnxifi_load(1, ONNXIFI_DUMMY_LIBRARY, "", &dummy_backend));
+			//Testing onnxGetBackendIDs
+				int numBackends = -1;
+				EXPECT_EQ(dummy_backend.functions[0](NULL, &numBackends), ONNXIFI_STATUS_SUCCESS);
```

Comment:
`dummy_backend.functions[0]` is cryptic. Why not use `dummy_backend.onnxGetBackendIDs`?

Reply:
(As I said I was just testing this way of using onnxGetBackendIDs. It is not ready for review.